packages/jmix-front-generator/src/building-blocks/stages/configuring/defaultConfigureGenerator.ts

```diff
@@ -20,12 +20,8 @@ export const defaultConfigureGenerator = <O extends CommonGenerationOptions>(tem
   // @ts-ignore this.env.adapter is missing in the typings
   gen.env.adapter
     .promptModule.registerPrompt('autocomplete',  AutocompletePrompt);
-  gen.registerTransformStream(createEjsRenameTransform());
-  gen.registerTransformStream(createFormatTransform());
-
-  if (options.answers || options.model) {
-    gen.conflicter.force = true;
-  }
```

Comment:
Why are we removing it? I think it's required in order to rewrite the files without confirmation.